tests/test_types_typeddict.py

```diff
@@ -33,6 +34,9 @@ def fixture_typed_dict(TypedDictAll):
     class TestTypedDict(TypedDictAll):
         foo: str
 
+    if sys.version_info < (3, 11) and TypedDictAll.__module__ == 'typing':
+        pytest.xfail('typing.TypedDict does not track required keys correctly on Python < 3.11')
```

Comment:
might be better to use `skip`?

The plan is to remove all `xfail`s before v2, and this code will remain I guess.